luigi/rpc.py

```diff
@@ -25,13 +25,33 @@
 import socket
 import time
 
-from luigi.six.moves.urllib.parse import urlencode
+from luigi.six.moves.urllib.parse import urlencode, ParseResult
 from luigi.six.moves.urllib.request import urlopen
 from luigi.six.moves.urllib.error import URLError
 
 from luigi import configuration
 from luigi.scheduler import PENDING, Scheduler
 
+
+HAS_UNIX_SOCKETS = True
+HAS_REQUESTS = True
+
+
+try:
+    import requests_unixsockets as requests
+except ImportError:
+    HAS_UNIX_SOCKETS = False
+    try:
+        import requests
+    except ImportError:
+        HAS_REQUESTS = False
+
+try:
+    from requests import exceptions as requests_exceptions
```

Comment:
Hmm, if you don't have it installed, won't you just get an exception like "no variable defined `requests_exceptions
Markdown supported `"? Can't you just do `from requests import exceptions as requests_exceptions` without try-catch inside the `fetch()` method?